### ISSUE TITLE
About CatVsDog Dataset

### ISSUE THREAD

**Issue by chaoqinhuang** [izikgo/AnomalyDetectionTransformations#2]
Hi, we download CatVsDog from Kaggle, and find no labels of test set. In your code, the file names of test data are also like cat.0.jpg. Can you share the labels? Or the link to download the labeled dataset. Thank you.

**Comment by chaoqinhuang** [comment]
Sorry. In your paper, we find that you split test set from training set. Maybe you can share how you split it.

**Comment by igolan89** [comment]
Just a simple random uniform split

**Comment by ghost** [comment]
Just a simple random uniform split
